code/__DEFINES/dcs/signals.dm

```diff
@@ -49,7 +49,7 @@
 // /atom signals
 #define COMSIG_ATOM_CREATED "atom_created"						///from base of atom/proc/Initialize(): sent any time a new atom is created
 #define COMSIG_PARENT_ATTACKBY "atom_attackby"			        ///from base of atom/attackby(): (/obj/item, /mob/living, params)
-	#define COMPONENT_NO_AFTERATTACK 1								//Return this in response if you don't want afterattack to be called
+	#define COMPONENT_NO_AFTERATTACK 1							//Return this in response if you don't want afterattack to be called
 #define COMSIG_ATOM_HULK_ATTACK "hulk_attack"					///from base of atom/attack_hulk(): (/mob/living/carbon/human)
 #define COMSIG_ATOM_ATTACK_ANIMAL "attack_animal"				///from base of atom/animal_attack(): (/mob/user)
 #define COMSIG_PARENT_EXAMINE "atom_examine"                    ///from base of atom/examine(): (/mob)
```

Comment:
whoever did this you are cringe.

Reply:
did what

Reply:
/// for defines (and didn't even do it right, as this would break the doc)

Reply:
this is pre-autodoc

Reply:
this still works as intended, the only issue is when you do this
```
    var/thing /// this gets merged into the thing below

/// this includes the codedoc comment above
/a/type/proc/foo()
```